### PR TITLE
HIVE-21260 : Hive 3 (onprem) -> 4(onprem): Hive replication failed due to postgres sql execution issue

### PR DESCRIPTION
**Pull request by maheshk114**
…

### CODE REVIEW

File: itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/WarehouseInstance.java [hunk 1]
```diff
@@ -154,6 +154,32 @@ private void initialize(String cmRoot, String externalTableWarehouseRoot, String
...
+    // Add the below mentioned dependency in metastore/pom.xml file. For postgres need to copy postgresql-42.2.1.jar to
+    // .m2//repository/postgresql/postgresql/9.3-1102.jdbc41/postgresql-9.3-1102.jdbc41.jar.
+    /*
+    <dependency>
+      <groupId>mysql</groupId>
+      <artifactId>mysql-connector-java</artifactId>
+      <version>8.0.15</version>
+    </dependency>
+
+    <dependency>
```
Comment (sankarh) [inline itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/WarehouseInstance.java]: Remove these irrelevant comments.

File: ql/src/java/org/apache/hadoop/hive/ql/parse/ReplicationSpec.java [hunk 2]
```diff
@@ -419,4 +419,11 @@ public boolean isMigratingToExternalTable() {
   public void setMigratingToExternalTable() {
     isMigratingToExternalTable = true;
   }
+
+  public static void setReplId(Map<String, String> srcParameter, Map<String, String> destParameter) {
```
Comment (sankarh) [inline ql/src/java/org/apache/hadoop/hive/ql/parse/ReplicationSpec.java]: This fix is irrelevant to the jira description. Can we make it another jira with scenario?

Comment (sankarh) [inline ql/src/java/org/apache/hadoop/hive/ql/parse/ReplicationSpec.java]: Can change name to copyLastReplId.

**Comment by maheshk114** [comment]
done

**Pull request closed without merging**